autoextract_poet/mixins.py

```diff
@@ -0,0 +1,19 @@
+from web_poet.mixins import ResponseShortcutsMixin
+from web_poet.page_inputs import ResponseData
+
+
+class AutoExtractResponseShortcutsMixin(ResponseShortcutsMixin):
+
+    @property
+    def response(self):
+        # FIXME: raise a custom Exception or create a separate method for errors
```

Comment:
This comment is outdated. Errors are not expected at this stage. The response should be well-formed here.

Reply:
The mixin was entirely removed for now.